libs/components/lists/src/lib/modules/repeater/repeater-item.component.ts

```diff
@@ -619,6 +676,21 @@ export class SkyRepeaterItemComponent
     this.#repeaterService.registerOrderChange();
   }
 
+  #updateContentInfo(): void {
+    this.#contextMenuResourceUnsubscribe.next();
+    if (this.itemNameOrDefault) {
+      this.#contentInfoProvider.patchInfo({
+        descriptor: this.itemNameOrDefault,
+      });
+    } else {
+      this.#contentInfoProvider.patchInfo({
+        descriptor: undefined,
+      });
+    }
+    // This is to void "Expression changed after checked" issues. `markForCheck` did not resolve the issue.
+    this.#changeDetector.detectChanges();
```

Comment:
there are two `detectChanges` I didn't really expect to see--here and in dropdown. Do we know _why_ this fixes it? Does it move something up in the change detection tree, move it to a different zone, etc? 

Did you try adding the mutationObserver to title after the view was initialized? I'm thinking the title doesn't pop in until after the change detection cycle has started and that is causing it to change after checked?

Edit: it does look like most of the places we use `SkyMutationObserverService` we call `create` in `afterViewInit`